lib/audit_history.rb

```diff
@@ -47,6 +47,7 @@ def property_class
   def find_association(id)
     return id unless association?
     begin
+      return NilAssociation.new if !id
```

Comment:
This can be moved out of the begin block, it can't ever raise an AR RecordNotFound :)